app/src/main/java/com/example/moneymanager/GamingRoomActivity.kt

```diff
@@ -16,15 +20,18 @@ class GamingRoomActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_gaming_room)
+        binding = ActivityGamingRoomBinding.inflate(layoutInflater)
+        val view: View = binding.root
```

Comment:
You can get rid of this line and pass `binding.root` as argument to `setContentView` directly.

Reply:
yes please, thank you!